File: client-java/sql/src/main/java/org/evomaster/client/java/sql/DbUnsupportedException.java
package org.evomaster.client.java.sql;

import org.evomaster.client.java.controller.api.dto.database.schema.DatabaseType;

public class DbUnsupportedException extends IllegalStateException{

    DbUnsupportedException(DatabaseType type){
        this(type.toString());
    }

    DbUnsupportedException(String type){
        super("Database type " + type + " is not supported");
    }

}

File: outdated-client-dotnet/src/EvoMaster.Controller/Controllers/db/DbUnsupportedException.cs
using System;
using EvoMaster.Controller.Api;

namespace EvoMaster.Controller.Controllers.db {
    public class DbUnsupportedException : InvalidOperationException {
        public DbUnsupportedException(DatabaseType type) : this(type.ToString()) { }

        public DbUnsupportedException(string type) : base("Database type " + type + " is not supported") { }
    }
}